podman-compose.dev.yml

```diff
@@ -0,0 +1,19 @@
+services:
+  serge:
```

Comment:
Whats the difference between this file and the current one?

Reply:
Adding :z at the end of lines 8 and 9 so podman can access the volumes without having to disable SELinux.

Reply:
@morpheus2448 You can add those to the existing compose file and delete this one

Reply:
ok